ydb/core/protos/config.proto

```diff
@@ -235,38 +261,12 @@ message TDomainsConfig {
         optional NKikimrCompaction.TCompactionPolicy Policy = 2;
     }
 
-    message TSecurityConfig {
-        optional bool EnforceUserTokenRequirement = 1 [default = false];
-        optional bool EnforceUserTokenCheckRequirement = 7 [default = false]; // Check if a token was specified // If not, or if the token was incorrect or access was denied, the request will be handled as if no token was provided
-        repeated string MonitoringAllowedSIDs = 2;
-        repeated string AdministrationAllowedSIDs = 3;
-        repeated string DefaultUserSIDs = 4;
-        optional string AllAuthenticatedUsers = 5;
-        repeated string ViewerAllowedSIDs = 6;
-        repeated string RegisterDynamicNodeAllowedSIDs = 8;
-
-        message TUser {
-            optional string Name = 1;
-            optional string Password = 2;
-        }
-
-        message TGroup {
-            optional string Name = 1;
-            repeated string Members = 2;
-        }
-
-        repeated TUser DefaultUsers = 15;
-        repeated TGroup DefaultGroups = 16;
-        repeated string DefaultAccess = 17;
-        optional string AllUsersGroup = 18;
-    }
-
     repeated TDomain Domain = 1;
     repeated TStateStorage StateStorage = 2;
     repeated TExecLevel ExecLevel = 3;
     repeated THiveConfig HiveConfig = 4;
     repeated TNamedCompactionPolicy NamedCompactionPolicy = 5;
-    optional TSecurityConfig SecurityConfig = 6;
```

Comment:
It's better to completely remove it here and add this field to TEphemeralConfig. It will help us avoid accidental use of wrong security config in new code.

Reply:
This one https://github.com/ydb-platform/ydb/blob/f3229908a211e80f79d1bec307cc0f486e447fab/ydb/library/yaml_config/protos/config.proto#L148
Check out transformations we already support in the same dir